### PR TITLE
Add warning on -novc

### PR DESCRIPTION
**Pull request by FrostTheFox**
Add a disclaimer that bad things can happen in -novc and that you won't receive support.
## Description

tl;dr log.warning('bad stuff can happen yo, no support 4 u')
## Motivation and Context

Make the dangers of -novc known
## How Has This Been Tested?

not tested, but i mean, c'mon, it's a log.warning.
## Screenshots (if appropriate):
n/a
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: runserver.py [hunk 1]
```diff
@@ -252,6 +252,12 @@ def main():
 
     set_log_and_verbosity(log)
 
+    if args.no_version_check:
```
Comment (friscoMad) [inline runserver.py]: Just warn on scanner instances not need to warn on -os instances, move it to can_start_scanning

Comment (FrostTheFox) [reply runserver.py]: I purposefully didn't put it in can_start_scanning because I was trying to keep that for checking stuff that would cause a sys.exit.

**Comment by FrostTheFox** [comment]
(2nd travis fail is because travis is having network issues)

File: runserver.py [hunk 2]
```diff
@@ -276,6 +276,13 @@ def main():
     if not args.no_server and not validate_assets(args):
         sys.exit(1)
 
+    if args.no_version_check:
```
Comment (pogo-excalibur) [inline runserver.py]: Could these be combined?
